docs/src/appendix/cheatcodes/l1_handler_execute.md

```diff
@@ -0,0 +1,76 @@
+# `l1_handler_execute`
+
+> `fn execute(self: L1Handler)`
+
+Executes a `#[l1_handler]` function to mock a
+[message](https://docs.starknet.io/documentation/architecture_and_concepts/L1-L2_Communication/messaging-mechanism/)
+arriving from Ethereum.
+
+```rust
+struct L1Handler {
+    contract_address: ContractAddress,
+    function_name: felt252,
+    from_address: felt252,
+    fee: u128,
+    payload: Span::<felt252>,
+}
+```
+
+where:
+
+- `contract_address` - The target contract address
+- `function_name` - Name of the `#[l1_handler]` function
+- `from_address` - Ethereum address of the contract that sends the message
+- `fee` - The fees paid on L1
+- `payload` - The message payload that may contain any Cairo data structure that can be serialized with
+[Serde](https://book.cairo-lang.org/appendix-03-derivable-traits.html?highlight=serde#serializing-with-serde)
+
+It is important to note that when executing the `l1_handler`,
+the `from_address` must always be checked as any L1 contract can call any L2 contract.
```

Comment:
I am not sure what does it mean "must be checked" 🤔

Reply:
In the sense that, not checking this can be an important security consideration. But perhaps you're right some people want to do open messaging. :P

Reply:
Just reworded with a `may be checked`. If it's not relevant at this level of the documentation, let me know and I'll remove it.

Reply:
I don't understand what exactly has to be checked

Reply:
The from_address argument, as you contract on L2 can receive L1HandlerTx from any L1 contract. As the official documentation always warn the user on this, I thought it was worthy to have a reminder here.

We can remove this sentence if you think it is not relevant in this context.

Reply:
I'd keep this sentence, imo it makes sense